keyboards/gmmk/numpad/readme.md

```diff
@@ -0,0 +1,29 @@
+# GMMK NUMPAD
+
+A 17 Key macropad made ans old by Glorious LLC, equiped with a WB32 microcontroller, with support for a rotary encoder.
+
+* Keyboard Maintainer: [GloriousThrall](https://github.com/GloriousThrall)
+* Hardware Supported: GMMK Numpad
+* Hardware Availability: [GloriousPCGaming.com](https://www.gloriousgaming.com/products/gmmk-numpad)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make gmmk/numpad:default
+
+Flashing example for this keyboard:
+
+    make gmmk/numpad:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the Num key and plug in the keyboard
+* **Keycode in layout**: Press the Num+Del key which is mapped to `QK_BOOT`
```

Comment:
Only 2 ways are mentioned here.  Does the PCB have any kind of a hardware reset button (for emergency usage, if some broken firmware version had been flashed)?

Also do the mentioned ways work when the board still has the Glorious Core firmware, or some other method needs to be used to flash the QMK firmware for the first time?

Reply:
There is a hardware reset button that's between the 2U `0` key and the `. Del` key on top of the PCB, which is accessible via a hole on the plate with a thin pin tool like a SIM card tool or paperclip.